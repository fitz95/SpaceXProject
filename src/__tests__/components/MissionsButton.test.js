import React from 'react';
import renderer from 'react-test-renderer';

import MissionsButton from 'src/components/MissionsButton';

describe('MissionsButton', () => {
  const onClick = () => {};

  it('renders correctly for reserved state', () => {
    const tree = renderer
      .create(<MissionsButton isReserved onClick={onClick} />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });

  it('renders correctly for not reserved state', () => {
    const tree = renderer
      .create(<MissionsButton isReserved={false} onClick={onClick} />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
