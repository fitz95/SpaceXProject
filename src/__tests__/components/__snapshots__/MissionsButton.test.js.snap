// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MissionsButton renders correctly for not reserved state 1`] = `
<button
  className="button joinMission"
  onClick={[Function]}
  type="button"
>
  Join Mission
</button>
`;

exports[`MissionsButton renders correctly for reserved state 1`] = `
<button
  className="button leaveMission"
  onClick={[Function]}
  type="button"
>
  Leave Mission
</button>
`;
